Maquetado/scss/titles.scss

```diff
@@ -0,0 +1,17 @@
+.title {  
+  font-size: $text-big;
+  position: relative;
+  margin: 40px 0 0 50px;
+  
+  &::before {
+    content: '';
+    background-image: url('./assets/triangle.svg');
+    background-repeat: no-repeat;
+    height: $title-decoration-height;
+    margin: -40px 0 0 -50px;
```

Comment:
you should never use negative margins, instead you should use `top`, `left`, `bottom` or `right` values to place the element properly